io.webtorrent.WebTorrent.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "io.webtorrent.WebTorrent",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "WebTorrent",
+  "rename-icon": "webtorrent-desktop",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "webtorrent",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x webtorrent.deb",
+        "tar -xf data.tar.gz",
+        "cp -r usr/* /app",
+        "cp -r opt/webtorrent-desktop/* /app/bin/",
+        "chmod +x /app/bin/WebTorrent",
+        "install -Dm644 io.webtorrent.WebTorrent.desktop /app/share/applications/io.webtorrent.WebTorrent.desktop",
+        "install -Dm644 io.webtorrent.WebTorrent.appdata.xml /app/share/appdata/io.webtorrent.WebTorrent.appdata.xml"
+      ],
+      "cleanup": [
+        "/share/doc"
+      ],
+      "sources": [
+        {
+          "type": "file",
```

Comment:
Could you support the i386 version as well using [only arches like this](https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk9/blob/master/org.freedesktop.Sdk.Extension.openjdk9.json#L32). I see there's an i386 binary.

https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.18.0/WebTorrent-v0.18.0-win-ia32.zip